code/turf/turf_flock.dm

```diff
@@ -269,25 +269,29 @@ turf/simulated/floor/feather/proc/bfs(turf/start)//breadth first search, made by
 /turf/simulated/wall/auto/feather
 	name = "weird glowing wall"
 	desc = "You can feel it thrumming and pulsing."
-	icon = 'icons/misc/featherzone.dmi'
-	icon_state = "0"
+	icon = 'icons/turf/walls_flock.dmi'
+	icon_state = "flock0"
+	mod = "flock"
 	health = 250
 	var/max_health = 250
-	flags = USEDELAY
+	flags = USEDELAY | ALWAYS_SOLID_FLUID | IS_PERSPECTIVE_FLUID
 	mat_appearances_to_ignore = list("steel", "gnesis")
 	mat_changename = FALSE
 	mat_changedesc = FALSE
-	connects_to = list(/turf/simulated/wall/auto/feather, /obj/machinery/door/feather)
-
+	connect_overlay = TRUE
+	connect_diagonal = TRUE
+	connects_to = list(/turf/simulated/wall/auto/feather, /obj/machinery/door, /obj/window)
+	connects_with_overlay = list(/obj/machinery/door, /obj/window)
```

Comment:
Do we want it connecting to non-Flock stuff? Felt it gave more of a contrast when it didn't and makes a bit more sense that it only connects to its own stuff, but if it looks nicer?

Reply:
Yeah, I considered having them separate but it looked shit so I changed it. The connectors round it off nicely.